src/components/MapView/Layers/raster-utils.ts

```diff
@@ -0,0 +1,290 @@
+import { point, Feature, MultiPolygon, Properties } from '@turf/helpers';
+import bbox from '@turf/bbox';
+import booleanPointInPolygon from '@turf/boolean-point-in-polygon';
+import * as GeoTIFF from 'geotiff';
+import { buffer } from 'd3-fetch';
+import { formatUrl } from '../../../utils/server-utils';
+
+export type TransformMatrix = [number, number, number, number, number, number];
+export type TypedArray =
+  | Uint8Array
+  | Uint16Array
+  | Uint32Array
+  | Int8Array
+  | Int16Array
+  | Int32Array
+  | Float32Array
+  | Float64Array;
+
+export interface Rasters extends Array<TypedArray> {
+  height: number;
+  width: number;
+}
+
+export type GeoJsonBoundary = Feature<MultiPolygon, Properties>;
+
+// GDAL style extent: xmin ymin xmax ymax
+export type Extent = [number, number, number, number];
+
+// Placeholder for Geotiff image (since library doesn't contain types)
```

Comment:
Should these be in type or customs?

Reply:
Could be either, really. In this case I put them here because I felt like the library was going to be basically contained to this file. But especially with a library that is used across the app, we'd probably want them to go in `custom`